File: eBotanika.Services/Database/Kategorija.cs
namespace eBotanika.Services.Database
{
    public partial class Kategorija
    {
        public Kategorija()
        {
            Biljke = new HashSet<Biljke>();
        }

        public int KategorijaID { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public virtual ICollection<Biljke> Biljke { get; set; }
    }
}

File: ebotanika_mobile/lib/models/kategorija.dart
class Kategorija {
  int kategorijaID;
  String naziv;
  String opis;

  Kategorija({
    required this.kategorijaID,
    required this.naziv,
    required this.opis,
  });

  factory Kategorija.fromJson(Map<String, dynamic> json) {
    return Kategorija(
      kategorijaID: json["kategorijaID"],
      naziv: json["naziv"],
      opis: json["opis"],
    );
  }

  Map<String, dynamic> toJson() => {
        "kategorijaID": kategorijaID,
        "naziv": naziv,
        "opis": opis,
      };
}
